JS/image.js

```diff
@@ -0,0 +1,45 @@
+// import { animals } from "./Pets";
+
+// let animals = [
+//   {
+//     id: 1,
+//     name: "Luna",
+//     imgSrc: "./assets/images/luna.jpg",
+//     type: "Cat",
+//     age: "4 years",
+//     character: ["Curious", "Affectionate", "Fast"],
+//     favToy: "Mouse",
+//     owner: "Gabriela",
+//     ownerUrl: "https://github.com/gabrysia694",
+//   },
+// ];
+
+
+// console.log(animals);
+let container = document.getElementById("container");
+/// Load the HTML template file using fetch
+fetch("../card.html")
+  .then((response) => response.text())
+  .then((html) => {
+    // Loop through the people array and replace the content of the template with the data
+    animals.forEach(function (animal) {
+      // Create a new div element and set its innerHTML to the template
+      let div = document.createElement("div");
+      div.innerHTML = html;
+
+      // Set the content of the template with the data
+      div.querySelector(".image").src = animal.imgSrc;
+      div.querySelector(".name-list").innerHTML = animal.name;
+      div.querySelector(".animal-list").innerHTML = animal.type;
+      div.querySelector(".age-list").innerHTML = animal.age;
+      div.querySelector(".character-1").innerHTML = animal.character[0];
+      div.querySelector(".character-2").innerHTML = animal.character[1];
+      div.querySelector(".character-3").innerHTML = animal.character[2];
```

Comment:
This code snippet needs to be modified. We haven't specified how many personality traits could be added; some people have added 3, other 5 like cat Snowey. You can put personality traits into an array and then use `.innerHTML` so you can pass any number of personality traits into the html element, and it will display them accordingly.